### ISSUE TITLE
Cancer gene rank function

### ISSUE THREAD

**Issue by dpolychr** [sigven/oncoEnrichR#34]
Hey @sigven, 

Hope all is well :-) 

I was wondering if it's possible to run cancer gene rank: https://sigven.github.io/oncoEnrichR/articles/cancer_gene_rank.html

as a stand-alone function for all genes? 

Interested in receiving cancer gene associations (pancan and HCC) for all genes

Thanks,
Dimitris

**Comment by sigven** [comment]
Hey Dimitris,

Hope all is well on your side too! Yeah, your suggestion makes sense, and this indeed doable, i'll get on to it next week.

kind regards,
Sigve

**Comment by sigven** [comment]
Hey Dimitris,

Check out the new version (1.5.2), it contains a dedicated cancer rank function. Looking at the underlying evidence data from the  Open Targets Platform, i see that some tumor sites/types have considerably more evidence than others, so for many sites, note that evidence will be scarce. 

Either way, let me know if the function works out for you, or if you have additional suggestions etc.

best,
Sigve

**Comment by dpolychr** [comment]
Thanks so much @sigven, very useful works really nicely :) 
I might have missed it but how is the no/non-significant cancer association with a specific tumor type determined? 
Maybe worth including this in the R Documentation

**Comment by sigven** [comment]
Good point @dpolychr! I will add this later on. Basically, to limit noisy evidence/association, there is a requirement that any association considered between a gene and a phenotype term (belonging to a tumor type/site), needs to be supported by at least two distinct types of evidence (e.g. data from literature mining and animal model experiments), and that the association score exceeds 0.05

**Comment by sigven** [comment]
Hi Dimitris,
Think maybe that current strategy is somewhat too stringent on retrieval of evidence from Open Targets, so looking into how to relax the thresholds quite a bit, enabling e.g. a capture of gene-disease evidence from single data types/sources (literature, animal models etc). The associations from OT is at the end of the day intended for ranking. Picking up subtle and emerging evidence is important too, I think.
